src/base/interfaces.ts

```diff
@@ -0,0 +1,37 @@
+export interface IRelationEntityReference {
+  id: string;
+  key: string;
+  columns: string[];
+}
+
+// TODO: Refactor
```

Comment:
Please be a bit more expressive here and give a few more pointers or link to an issue. If we don't do it soon then we will forget what we meant with "refactor".